lib/manager/cdn/extract.ts

```diff
@@ -0,0 +1,34 @@
+import { PackageFile, PackageDependency } from '../common';
+import { DATASOURCE_CDNJS } from '../../constants/data-binary-source';
+
+export function extractPackageFile(content: string): PackageFile {
+  const deps: PackageDependency[] = [];
+
+  const regex = /\/\/cdnjs\.cloudflare\.com\/ajax\/libs\/(?<depName>[^/]+?)\/(?<currentValue>[^/]+?)\/(?<asset>[-_.a-zA-Z0-9]+)/;
```

Comment:
This is fine for an initial implementation, but we should probably be parsing HTML and only updating `<script>` tags.

Reply:
I just had another though. If any sites are using [Subresource integrity](https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity) this implementation will break the site.

It appears that the CDNJS API doesn't provide file hashes for hosted dependencies.

Reply:
See cdnjs/new-website#246